src/db/models/cipher.rs

```diff
@@ -382,6 +382,11 @@ impl Cipher {
             // the "Read Only" or "Hide Passwords" restrictions for the user.
             json_object["edit"] = json!(!read_only);
             json_object["viewPassword"] = json!(!hide_passwords);
+            // The new key used by clients since v2025.6.1
```

Comment:
LGTM. Only this comment bugs me a bit:

```suggestion
            // The new key used by clients since v2025.6.0
```

For most clients it's `v2025.6.0`, cf. https://github.com/dani-garcia/vaultwarden/discussions/5996#discussioncomment-13610267 and https://github.com/bitwarden/clients/commit/b5bddd0b06b9a05dedd02197d9aaa7fd0ff2b3c7 :nerd_face:

Reply:
Ah, it's since .0 then,  only the web-vault since .1